dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebProfileEARtest.java

```diff
@@ -74,7 +74,6 @@ public void testEARdeployment() throws Exception {
 
         server.setCheckpoint(CheckpointPhase.APPLICATIONS, false, null);
         server.startServer();
-        server.stopServer();
```

Comment:
How is this related to the bells checkpoint support?  Looks like it is correct that the `stopServer` should not be needed here in this test though, so not requesting you change the PR for this.

Reply:
Yes this is not related to bells. I came accross this while updating the bootstrap properties.